utils/__tests__/getDataUtils.spec.ts

```diff
@@ -0,0 +1,158 @@
+import {
+  parseCategory,
+  getCategories,
+  getAllPhrases,
+  getKidsCategory,
+  getPhraseById,
+  fetchRawDictionary,
+  fetchDictionary,
+  CategoryDataObject,
+  DictionaryDataObject,
+  PhraseDataObject,
+  Phrase,
+} from '../getDataUtils';
+import { CountryVariant } from '../locales';
+import fetch, { Response } from 'node-fetch';
+
+jest.mock('node-fetch');
+
+// Create mock data for testing
+
+const mockCountryVariantCZ = 'cz' as CountryVariant;
```

Comment:
This is a bad use of `as`, generally try to avoid `as` if you can. 

This should be `const mockCountryVariantCZ: CountryVariant = 'cz';` which would make TS tell you that `cz` is not a valid CountryVariant, `cs` is :)

Reply:
Fixed cz vs cs :)